packages/manager-components/src/components/layout/dashboard/dashboard.spec.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-return-await */
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { render, waitFor, screen } from '../../../utils/test.provider';
```

Comment:
why would you do that ? you have to import those directly from testing library

Reply:
Yes, I delete this useless import and export inside test.provider 👍